source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/utils/wrappers/skrl.py

```diff
@@ -26,23 +27,22 @@
 # needed to import for type hinting: Agent | list[Agent]
 from __future__ import annotations
 
-from skrl.envs.wrappers.torch import wrap_env
-from skrl.resources.preprocessors.torch import RunningStandardScaler  # noqa: F401
-from skrl.resources.schedulers.torch import KLAdaptiveLR  # noqa: F401
-from skrl.utils.model_instantiators.torch import Shape  # noqa: F401
-
 from omni.isaac.lab.envs import DirectRLEnv, ManagerBasedRLEnv
 
 """
 Configuration Parser.
 """
 
 
-def process_skrl_cfg(cfg: dict) -> dict:
+def process_skrl_cfg(cfg: dict, ml_framework: str = "torch") -> dict:
```

Comment:
You can make a literal to make the choices explicit

```suggestion
def process_skrl_cfg(cfg: dict, ml_framework: Literal["torch", "jax"] = "torch") -> dict:
```